grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -677,15 +699,15 @@ class ReportService implements ApplicationContextAware {
             def qtyOnHand = it.qtyOnHand ? it.qtyOnHand.toInteger() : 0
             def qtyOrderedNotShipped = it.qtyOrderedNotShipped ? it.qtyOrderedNotShipped.toInteger() : 0
             def qtyShippedNotReceived = it.qtyShippedNotReceived ? it.qtyShippedNotReceived : 0
-            Product product = Product.findByProductCode(it.productCode)
+            def displayNameWithLocaleCode = "${it.productName}${it.displayName ? " (${locale?.toUpperCase()}: ${it.displayName})" : ''}"
```

Comment:
For now I don't mind it, but be aware, that if we ever consider changing the `getDisplayNameWithLocaleCode()` method in the `Product` domain, we'd have to cover this place as well.
I think for now it could stay, but if we once again have to do it anywhere else, I'd consider creating a static method in the `Product` domain which would take the arguments: `name`, `displayName` and `locale` and place it below the already existing method, so we'd always remember to change both.